runtime/bcutil/defineclass.c

```diff
@@ -511,8 +511,8 @@ internalLoadROMClass(J9VMThread * vmThread, J9LoadROMClassData *loadData, J9Tran
 		translationFlags |= stripFlags;
 	}
 
-	if ((0 == (loadData->options & J9_FINDCLASS_FLAG_UNSAFE)) &&
-		(0 != (vm->runtimeFlags & J9_RUNTIME_VERIFY))) {
+	/* Enable the static verification on the unsafe classes */
+	if (J9_ARE_ANY_BITS_SET(vm->runtimeFlags,J9_RUNTIME_VERIFY)) {
```

Comment:
The comment can be deleted here as it doesn't match the intention of the code anymore.  Also, nitpick on space between `, J9_RUNTIME....`
```suggestion
	if (J9_ARE_ANY_BITS_SET(vm->runtimeFlags, J9_RUNTIME_VERIFY)) {
```

Reply:
Updated against the suggestion above.